Core/include/Acts/Navigation/DetectorNavigator.hpp

```diff
@@ -174,7 +183,8 @@ class DetectorNavigator {
                                     state.surfaceCandidateIndex)
                                 << " out of " << state.surfaceCandidates.size()
                                 << " surfaces remain to try.");
-    // Take the surface
+
+    // Take the surface and mark it as a portal or a surface
```

Comment:
```suggestion
    // Take the surface
```
nitpick: looks like this is not true anymore?